kotlin-emotion/src/main/kotlin/emotion/css/keyframes.ext.kt

```diff
@@ -0,0 +1,21 @@
+@file:Suppress(
+    "DECLARATION_CANT_BE_INLINED",
```

Comment:
Is it legal?

Reply:
It was made by analogy with `RuleBuilder`

Reply:
Cancelled